modules/features/clip/src/main/kotlin/au/com/shiftyjelly/pocketcasts/clip/ShareClipFragment.kt

```diff
@@ -77,7 +77,7 @@ class ShareClipFragment : BaseDialogFragment() {
         container: ViewGroup?,
         savedInstanceState: Bundle?,
     ) = ComposeView(requireActivity()).apply {
-        val listener = ShareClipViewModelListener(this@ShareClipFragment, viewModel, clipAnalytics.analyticsTracker)
+        val listener = ShareClipViewModelListener(this@ShareClipFragment, viewModel)
```

Comment:
Checking figma design, I think the play/pause button should be moved a bit to right:

| Current Implementation | Figma |
|--------|--------|
| ![Screenshot_20240723_124153_Pocket Debug](https://github.com/user-attachments/assets/1729fc49-4a3f-40cd-bb42-7d2d51e0c998) | ![Screenshot_20240723_124153_Pocket Debug](https://github.com/user-attachments/assets/b10b3189-4bb2-479e-a400-865b0e0f88cd) |

Reply:
I think this is a minor thing, but I am just checking

Reply:
How do you compare that? Those are different states. One on the left has clip selector in the middle of the timeline. One on the right has it at the very start of it.